packages/cadl-vs/README.md

```diff
@@ -1,24 +0,0 @@
-# Cadl Language Support for Visual Studio
```

Comment:
For some reason a fair number of files are showing as deleted + added instead of renamed + modified, making it hard to review. :(

Reply:
Yeah

Reply:
I'm not sure, but it's possible that when the commits are squashed, git will have an easier time detecting the rename. I will try this in a branch and make a local pass.

Reply:
As we discussed. The PR is now separated by commits which should show clearly the change better. Yup We don't want to squash merge.